credentials/tls/certprovider/pemfile/watcher.go

```diff
@@ -78,6 +79,10 @@ func (o Options) canonical() []byte {
 }
 
 func (o Options) validate() error {
+	// Guard against SPIFFE bundle map usage
```

Comment:
My reading of this sentence from the gRFC is that we don't even read the new field `spiffe_trust_bundle_map_file` from the bootstrap. So, this might need to be handled in `certprovider/pemfile/builder.go` in `pluginConfigFromJSON` instead of here. Because by the time we get here, we have already read the field from the bootstrap JSON.

"""
The xDS functionality will be guarded via the GRPC_EXPERIMENTAL_XDS_MTLS_SPIFFE environment variable. The new bootstrap field will not be read if this env var is unset. This env var guard will be removed when the feature has passed interop tests.
"""